.github/workflows/test.yml

```diff
@@ -39,3 +39,9 @@ jobs:
         upload-core-artifacts: true
         upload-example-artifacts: true
         upload-storybook-artifact: true
+
+  unicorn:
+    needs:
+      - test
```

Comment:
Should we want to provide an unicorn deployment independenly from the `test` job?

Reply:
Yea, it would be nice if we could also trigger it manually. Wouldn't it be just adding `manual` to the triggers?

Reply:
Should the `manual` trigger on `test,yml` or `deploy-unicorn,yml`? In case of  `deploy-unicorn,yml` we need to add a build step for the binaries or add an input for the specific `workflow_id`.

Reply:
You are right, that seems to be kinda complicated. I think it's fine then, we don't need to able to run an independent job